js/jquery.terminal.js

```diff
@@ -1357,6 +1358,8 @@
                     } else {
                         if (command !== '' && position > 0) {
                             self['delete'](-1);
+                            var str = $('.clipboard').val()
+                            $('.clipboard').val(str.substring(0, str.length - 1));
```

Comment:
You can remove the `if (touch) {` from [fake_mobile_entry](https://github.com/jcubic/jquery.terminal/blob/master/js/jquery.terminal-src.js#L934] change the name to `fix_textarea` and change all occurences of that function then you will be able to remove that substring from backspace and it should work. I'll be happy to merge if you do.